### ISSUE TITLE
Create dashboards for Routers

### ISSUE THREAD

**Issue by jlpedrosa** [vaerh/mikrotik-prom-exporter#6]
If we want this project to be easy to use we also need to create the Dashboards and publish it on: https://grafana.com/grafana/dashboards/  Potentially we can do the automatic integration with grafana operator on to import them automatically.

**Comment by vaerh** [comment]
I don't see a big problem here either. After some alpha-version we will create a dashboard, here at your choice.

**Comment by jlpedrosa** [comment]
yeah, just creating the "backlog", if you think of anything pending, let's add it to the list..
